app/javascript/packs/components/forms/LocationInput.jsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import parse from "autosuggest-highlight/parse";
+import throttle from "lodash/throttle";
+import StyledTextField from "./StyledTextField";
+
+function loadScript(src, position, id) {
+  if (!position) {
+    return;
+  }
+
+  const script = document.createElement("script");
+  script.setAttribute("async", "");
```

Comment:
I think you can use `removeAttribute` if you want?

Reply:
I believe this is setting a boolean attribute, like this `<script async src="...`

Reply:
Ah ok.